website/source/docs/internals/providers/new-data-source.html.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: "docs"
+page_title: "Creating Data Sources"
+sidebar_current: "docs-internals-provider-guide-new-data-source"
+description: |-
+  How to get started adding a data source to an existing provider.
+---
+
+# Creating Data Sources
+
+Data sources are components of providers that pull information from outside of
```

Comment:
This isn't necessarily true. Data sources pull data from external sources, but those externals sources could be managed by Terraform too. This makes it sound mutually exclusive.